themes/socialbase/templates/file/image-widget.html.twig

```diff
@@ -41,7 +41,7 @@
   {% if in_post %}
     {# Render a "fake" remove button that triggers the actual (hidden) remove button. #}
     <div class="hidden">{{ data.remove_button }}</div>
-    <button type="button" id="post-photo-remove" class="btn--post-remove-image">
+    <button type="button" id="post-photo-remove" class="btn--post-remove-image" aria-hidden="true">
```

Comment:
@agami4 have you tested this with a screenreader?

This change makes it no longer possible to remove an image using a screenreader. We should not hide the button entirely but only the SVG icon.

We can also move the `.sr-only` text as `<title>` into the SVG element. This would accomplish the same result I think.

Reply:
@Kingdutch, I've removed the `aria-hidden` attribute and moved help text to the title on the SVG